packages/drivers/form/src/form.js

```diff
@@ -0,0 +1,35 @@
+import { types } from 'k-ramel'
+import actions from './actions'
+import selectors from './selectors'
+
+export default ({
+  path = 'form',
+  getState = store => store.form,
+}) => ({
+  getReducer: () => ({
+    path,
+    reducer: {
+      values: types.keyValue(),
+      errors: types.keyValue(),
+    },
+  }),
+  getDriver: store => (form) => {
+    const state = getState(store)
+    const { set, reset, onChange, destroy } = actions(state)(form)
```

Comment:
why don't you write actions on values/errors into action file ?
then here
```
return ({
  ...actions(state)(form),
  ...selectors(state)(form),
})
````

and then you can activate back the eslint rule